h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/rabit/RabitTrackerH2O.java

```diff
@@ -213,13 +213,16 @@ public int waitFor(long timeout) {
                 this.trackerThread.join(timeout);
             } catch (InterruptedException e) {
                 Log.debug("Rabit tracker thread got suddenly interrupted.", e);
+                Thread.currentThread().interrupt();
             }
         }
         return 0;
     }
 
     @Override
     public void uncaughtException(Thread t, Throwable e) {
+        Log.err("Uncaught exception occurred on Rabit tracker thread " + t.getName(), e);
+        Log.err(e);
```

Comment:
second log looks useless, doesn't it?

Reply:
the first one does not log the stacktrace, so its not.

Reply:
oh! I'm so much used to `commons.logging` and/or `log4j` that I didn't noticed that our logger was incompatible with those: such a shame...

Reply:
true that brother. https://0xdata.atlassian.net/browse/PUBDEV-7046